packages/common-all/src/abTests.ts

```diff
@@ -89,6 +89,28 @@ export const _2022_09_VIDEO_TUTORIAL_TEST = new ABTest(
   ]
 );
 
+/**
+ * Experiment to test highlights in quickpick labels
+ */
+export enum QuickPickLabelHighlightTestGroups {
+  "none" = "none",
+  "label" = "label",
+}
+
+export const _2022_09_QUICKPICK_LABEL_HIGHLIGHT_TEST = new ABTest(
```

Comment:
nit: can we have somewhere in the test name that this concerns the apply-template row in the quickpick? Otherwise it's harder to know what experiemnt 'quickpick-label' is referring to